pkgs/applications/audio/aaxtomp3/default.nix

```
@@ -36,17 +27,51 @@ stdenv.mkDerivation rec {
     hash = "sha256-7a9ZVvobWH/gPxa3cFiPL+vlu8h1Dxtcq0trm3HzlQg=";
   };
 
-  dontBuild = false;
+  # use whitespace to show osh arithmetic is not file redirection
+  # see: https://github.com/oilshell/oil/issues/1446
+  patches = [./osh.patch];
 
-  nativeBuildInputs = [ makeWrapper ];
+  postPatch = ''
+    substituteInPlace AAXtoMP3 \
+      --replace 'AAXtoMP3' 'aaxtomp3'
+    substituteInPlace interactiveAAXtoMP3 \
+      --replace 'AAXtoMP3' 'aaxtomp3' \
+      --replace 'call="./aaxtomp3"' 'call="${placeholder "out"}/aaxtomp3"'
```

```suggestion
      --replace 'call="./aaxtomp3"' 'call="$AAXTOMP3"'
```

Sorry. I misled you before. I wasn't thinking enough steps ahead, and the call behavior is dynamic enough that it's outside of what resholve can ~comfortably handle.

resholve is using a two-phase build to accommodate the occasional shell package that does its own dynamic build-time code generation/adaptation/templating/etc. That means the `out` placeholder here is for the inner (unresholved) output.

It's ugly, but instead we can patch a variable into this declaration and then let resholve fix this variable to the right path down below.